taskcluster/kinds/client/kind.yml

```diff
@@ -21,7 +19,10 @@ task-defaults:
 tasks:
   nodejs:
     description: nodejs client tests
-    docker-image: node:{node_version}
+    worker:
+      docker-image: node:20.16.0
```

Comment:
do we need a way to bump this version number when we update our node version?

Reply:
Yep! I'm going to work on the generators after the main hunk of work is complete so that these will get updated with `yarn generate`.